config11/config11.c

```diff
@@ -57,15 +58,20 @@ char *namtab[RANK_LNT] = {
 
 int main (int argc, char *argv[])
 {
+(void) argc;
+(void) argv;
+
 for ( ;; ) {
     for (i = 0; i < RANK_LNT; i++) numctl[i] = 0;
     printf ("Enter configuration data\n");
     for ( ;; ) {
 	printf ("Name:\t");
-	if (gets (inp) == NULL) return 0;
+	if (fgets (inp, sizeof (inp), stdin) == NULL) return 0;
+	for (cp = inp; *cp != 0; cp++) if (*cp == '\r' || *cp == '\n') *cp = '\0';
 	if (*inp == 0) break;
 	for (cp = inp; *cp != 0; cp++) *cp = toupper (*cp);
 	for (rank = 0; rank < RANK_LNT; rank++) {
+		printf ("cmp: \"%s\" == \"%s\"?\n", inp, namtab[rank]);

```

Comment:
DId you mean to include this debug code in the PR?

Reply:
No, I don't think so. Thanks for catching it!

Change and force-push? Another commit to remove that line again /revert that part?